models/AbstractStorage.cfc

```diff
@@ -41,9 +41,9 @@ component
 		}
 
 		// else, produce it
-		lock name="getOrSet.#variables.lockName#.#arguments.name#" type="exclusive" timeout="#variables.timeout#" throwonTimeout="true"{
+		lock name="getOrSet.#variables.lockName#.#arguments.name#" type="exclusive" timeout="#variables.lockTimeout#" throwonTimeout="true"{
```

Comment:
Good catch, but I think that this has another bug. Since the abstract class is using a `variables.lockTimeout` then we need to refactor into the class that property:

```
/**
	* Lock Timeout
	*/
	property name="lockTimeout" default="20" type="numeric";
```

So we would have to remove that property from the concrete classes and leave it on the abstract one. We also should initialize that property in the pseudo constroctur so ALL concrete classes have a default value declared.